Portfolio/index.html

```diff
@@ -9,5 +9,50 @@
   <body style="margin: 0; padding: 0">
```

Comment:
The <center> tag is not supported in HTML5. Use CSS to center align the heading instead.